CPAProxy.podspec

```diff
@@ -1,23 +1,34 @@
 Pod::Spec.new do |s|
   s.name            = "CPAProxy"
-  s.version         = "1.1.0"
+  s.version         = "1.2.0"
   s.summary         = "CPAProxy is a thin Objective-C wrapper around Tor."
   s.author          = "Claudiu-Vlad Ursache <claudiu.vlad.ursache@gmail.com>"
 
   s.homepage        = "https://github.com/ursachec/CPAProxy"
   s.license         = { :type => 'MIT', :file => 'LICENSE.md' }
-  s.source          = { :git => "https://github.com/chrisballinger/CPAProxy.git", :branch => "podspec"}
+  s.source          = { :git => "https://github.com/ursachec/CPAProxy.git", :branch => "master"}
```

Comment:
This should point to YND fork and not to `master` but tagged version -- I'll update that.